### ISSUE TITLE
Can't use CppAbi enum in user code

### ISSUE THREAD

**Issue by jspohr** [mono/CppSharp#1110]
##### Brief Description
Using version 0.8.20 from NuGet, I can't use values from the CppAbi enum to configure the parser. 
The following code does not compile:

```
namespace CppSharpTest {
  class Program {
    static void Main(string[] args) {
      var abi = CppSharp.Parser.AST.CppAbi.Itanium;
    }
  }
}
```

It produces the following error message:

```
error CS0433: The type 'CppAbi' exists in both 'CppSharp.Parser.CLI, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null' and 'CppSharp.Parser.CSharp, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null'
```
It doesn't matter which enum value I use, as the error refers to the enum class itself. When switching to 0.8.19, everything works fine.

OS: Windows 10
Visual Studio 2015

packages.config:
```
<?xml version="1.0" encoding="utf-8"?>
<packages>
  <package id="Baseclass.Contrib.Nuget.Output" version="2.4.3" targetFramework="net452" />
  <package id="CppSharp" version="0.8.20" targetFramework="net452" developmentDependency="true" />
</packages>
```

##### Used settings
Target: MSVC
New project of type "C# Console Application", changed .NET TargetFrameworkVersion to 4.6.1, and PlatformTarget to x64. All other settings are defaults.

**Comment by tritao** [comment]
@ddobrev Any idea?

**Comment by ddobrev** [comment]
Fixed in release 0.8.21.
